src/redux/reducers/currentWeather.js

```diff
@@ -0,0 +1,46 @@
+import {
+  SET_CURRENT_SUCCESS,
+  SET_CURRENT_FAIL,
+  SET_CURRENT_LOADING,
+  SET_CURRENT_LOADED
+} from '../actions/currentWeather';
+
+const initialState = {
+  isLoading: false,
+  current: null,
+  error: []
```

Comment:
You sure error should be an array? I mean at least name it errors probably... what do you think ?

Reply:
I'm keeping all errors there, so it should be an array, but you're totally right about it having to be plural